README.md

```diff
@@ -4,12 +4,28 @@ A open source screen reader for android.
 ## Build
 In development, choose the *debug build type*, because for the *release build type*, it's necessary to create a [keystore](https://developer.android.com/training/articles/keystore) and specify its properties in the `keystore.properties` file (use `keystore.properties.example` as a base).
 
-### IDE
-To build you need [Android Studio](https://developer.android.com/studio/releases) (Flamingo or later).
+### Android Studio
 
-1. Clone the repository to your local machine.
-2. Open the project in the IDE.
-3. Run the **app** module.
+1. Download and install [Android Studio](https://developer.android.com/studio).
```

Comment:
Regarding previous comment, we can drop previous note and write `1. Download and install [the latest version of Android Studio](https://developer.android.com/studio).`